lib/rmt/lockfile.rb

```diff
@@ -4,13 +4,11 @@ class RMT::Lockfile
 
   class << self
     def create_file
-      if File.exist?(LOCKFILE_LOCATION)
-        raise RMT::Lockfile::ExecutionLockedError
-      else
-        dirname = File.dirname(LOCKFILE_LOCATION)
-        FileUtils.mkdir_p(dirname) unless File.directory?(dirname)
-        File.open(LOCKFILE_LOCATION, 'w') { |f| f.write(Process.pid) }
-      end
+      dirname = File.dirname(LOCKFILE_LOCATION)
+      FileUtils.mkdir_p(dirname) unless File.directory?(dirname)
+
+      f = File.open(RMT::Lockfile::LOCKFILE_LOCATION, File::RDWR | File::CREAT)
```

Comment:
Once the method returns, `f` isn't referenced anymore and gets freed by the garbage collector, meaning that the file is closed and the lock is released.